apps/dashboard/pages/donations.tsx

```diff
@@ -7,25 +7,39 @@ import { useRouter } from 'next/router'
 import { useStrapiRequest } from '@wsvvrijheid/services'
 import { ssrTranslations } from '@wsvvrijheid/services/ssrTranslations'
 import { Donation, Sort, StrapiLocale } from '@wsvvrijheid/types'
+import { MonthPicker } from '@wsvvrijheid/ui'
 import { AdminLayout, DataTable, PageHeader, useColumns } from '@wsvvrijheid/ui'
+import { RangeParams } from '@wsvvrijheid/ui/src/components/MonthPicker/types'
 
 type PageProps = InferGetStaticPropsType<typeof getStaticProps>
 
 const DonationsPage: FC<PageProps> = ({ seo }) => {
   const [currentPage, setCurrentPage] = useState<number>(1)
   const [searchTerm, setSearchTerm] = useState<string>()
+  const [date, setDate] = useState<RangeParams>()
 
   const [sort, setSort] = useState<Sort | undefined>(['createdAt:desc'])
 
   const { locale } = useRouter()
   const columns = useColumns<Donation>()
 
+  const startDate =
+    date && new Date(date.startYear, date.startMonth).toISOString()
+
+  const endDate = date && new Date(date.endYear, date.endMonth).toISOString()
+
   const donationsQuery = useStrapiRequest<Donation>({
     endpoint: 'donates',
     page: currentPage || 1,
     pageSize: 50,
     filters: {
       ...(searchTerm && { email: { $containsi: searchTerm } }),
+      ...(date && {
+        createdAt: {
+          $gte: startDate,
+          $lte: endDate,
```

Comment:
```suggestion
          $lt: endDate,
```

`lestThanEqual` will include the first day of the next month.